core/components/atoms/alert/alert.js

```diff
@@ -119,7 +119,12 @@ Alert.Element = styled.div`
     right: 0;
     top: 0;
     color: ${props => colors.alert[props.type].text};
+    opacity: .2;
     padding: ${spacing.small} ${spacing.small};
+    &:hover {
+      color: rgba(0,0,0,0);
```

Comment:
Hey @mrewers! Thanks for your contribution. Here you are setting the cross not to be visible on hover. That is not the behavor we would want for this state.

![image](https://user-images.githubusercontent.com/4152942/46438726-feb06500-c734-11e8-9f72-92f4e20431f4.png)